pkg/go.mod

```diff
@@ -57,7 +57,7 @@ require (
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	google.golang.org/api v0.74.0
 	google.golang.org/genproto v0.0.0-20220405205423-9d709892a2bf
-	google.golang.org/grpc v1.47.0
```

Comment:
Hmm.. interesting `go mod tidy` resulted in all these additional updates, probably because go.mod in pulumi-java now depends on higher versions of grpc and such (we've been accepting dependabot-initiated updates when those pass tests).